src/Components/RecipeFood.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+const RecipeFood = (props) => {
+  const { detailsRecipe: { strMeal, strMealThumb, strCategory } } = props;
+  return (
+    <div>
+      <h2 data-testid="recipe-title">{strMeal}</h2>
+      <img
+        src={strMealThumb}
+        alt={strMeal}
+        data-testid="recipe-photo"
+        width="200"
+      />
+      <p data-testid="recipe-category">{strCategory}</p>
+    </div>
+  );
+};
```

Comment:
acho que se não ŕecisar de mais nada aqui da pra fazer o retorno implicito, e desestruturar no parametro, mais não sei se isso é melhor ou não kkkkkk.
mais ou menos isso kkk, mais da uma olhada primeiro pra ver qual fica melhor.
```suggestion
const RecipeFood = ({ detailsRecipe: { strMeal, strMealThumb, strCategory } }) => (
    <div>
      <h2 data-testid="recipe-title">{strMeal}</h2>
      <img
        src={strMealThumb}
        alt={strMeal}
        data-testid="recipe-photo"
        width="200"
      />
      <p data-testid="recipe-category">{strCategory}</p>
    </div>
);

```